index.html

```diff
@@ -1,63 +1,205 @@
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" type="text/css" href="./style.css" />
-    <title>My First Portfolio</title>
-  </head>
-  <body>
-    <header>
-      <div id="logo-wraper">
-        <a href="#" id="logo">My Logo</a>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" type="text/css" href="./style.css" />
+  <title>My First Portfolio</title>
+</head>
+
+<body>
+  <header>
+    <div id="logo-wraper">
+      <a href="#" id="logo" alt="A logo that says My Logo">My Logo</a>
+    </div>
+    <nav id="app-bar">
+      <div id="hamburger-button">
+        <img alt="Menu's icon" src="./visuals/hamburger-icon.png" />
       </div>
-      <nav id="app-bar">
-        <div id="hamburger-button">
-          <img src="./visuals/hamburger-icon.png" />
-        </div>
-      </nav>
-    </header>
-    <section id="headline">
-      <div id="headline-container">
-        <h1 class="title element" id="headline-title">
-          I’m Javier. Glad to see you!
+    </nav>
+  </header>
+  <section id="headline">
+    <div id="headline-container">
+      <h1 class="title element" id="headline-title">
+        I’m Javier. Glad to see you!
+      </h1>
+      <p class="paragraph element">
+        I’m a software developer! I can help you build a product, feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact
+        me.
+      </p>
+      <h2 class="supporting-text element">LET’S CONNECT</h2>
+      <ul class="element" id="social-media-container">
+        <l1>
+          <a href="#">
+            <img class="icon" alt="Twitter's icon" src="./visuals/twitter-icon.png" />
+          </a>
+        </l1>
+        <l1>
+          <a href="#">
+            <img class="icon" alt="LinkedIn's icon" src="./visuals/linkedin-icon.png" />
+          </a>
+        </l1>
+        <l1>
+          <a href="#">
+            <img class="icon" alt="Medium's icon" src="./visuals/medium-icon.png" />
+          </a>
+        </l1>
+        <l1>
+          <a href="#">
+            <img class="icon" alt="GitHub's icon" src="./visuals/github-icon.png" />
+          </a>
+        </l1>
+        <l1>
+          <a href="#">
+            <img class="icon" alt="AngelList's icon" src="./visuals/angellist-icon.png" />
+          </a>
+        </l1>
+      </ul>
+    </div>
+  </section>
+  <section id="works">
+    <div class="work-card">
+      <img class="work-snapshot element" src="./visuals/tonic-snapshot.png" />
+      <div class="work-text-block element">
+        <h1 class="title" id="headline-title">
+          Tonic
         </h1>
```

Comment:
Please consider using a heading tag other than the h1 tag here as you have already used it in the headline section. The reason is there must be only one h1 tag through so the browser can easily understand what you want to show users when they search.


## NOTE:-

This problem exists in all the cards so please consider using other heading tags like h2 or other for all.